pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -286,27 +312,22 @@ def connection_test(context):
 
 
 @connection_group.command('save', options_metavar=CMD_OPTS_TXT)
-@click.option('-n', '--name', type=str, metavar="NAME",
-              default=None,
-              help='Name for the new WBEM connection definition.')
```

Comment:
Why is --name removed? It can very well be used to use a previously defined connection defin ition as input for the new connection definition.

Reply:
It duplicates the same thing in add.  Note that I changed the name of that param in add to --input-name. For some reason, click did not like an argument and option with the same name.

In any case I included that option.
DONE

Reply:
I don't like the option name --input-name at all. In all other commands, the option name is --name for the same thing. We need to find another solution for the variable name collision.

I think the collision only exists between subcommand option --name and subcommand argument NAME, but not involving general option --name.

That is related to the question I stumbled across yesterday when reviewing this PR: Why does connection add have command options for specifying the connection when the same options already exist as general options? I think add could use the general options for what it does. The only thing we need to watch out for is the current connection, which we don't want to change, but I think when used in interactive mode, a connection add has its options in another context than the outermost context. We can try that out to make sure.

Reply:
DONE

Reply:
Ok. The description of the command still mentions --input-name, though. Also, it only saves the current connection, not "a connection".

Change:

    Save a connection to a new WBEM connection definition named NAME.

    Create a new WBEM connection definition in the connections file from the
    current connection or the connection definition in --input-name anamed
    NAME.

To:

    Save the current connection to a new WBEM connection definition.

    Create a new WBEM connection definition named NAME in the connections
    file from the current connection .

Reply:
TODO